.buildkite/pipeline.yml

```diff
@@ -79,7 +79,7 @@ steps:
           context: "UI Tests (iPhone)"
 
   - label: "🔬 UI Tests (iPad)"
-    command: .buildkite/commands/run-ui-tests.sh UITests "iPad Air (4th generation)" 15.0
+    command: .buildkite/commands/run-ui-tests.sh UITests "iPad Air (5th generation)" 15.0
```

Comment:
Some of the iPad tests were failing in CI but were not reproducible locally after this change, from investigation found that it was because, in CI, the iPad tests were running on iPhone 8 instead of iPad. See screenshot from one of the failing builds in this branch:

<img width="1046" alt="image" src="https://user-images.githubusercontent.com/17252150/188560722-e3bb313a-3288-48c2-8698-abe4d196f122.png">

Reply:
@jostnes - FYI, there's #7662 that updates exactly the same. I think it's maybe worth cancelling your 8821b76c2f5a72d4915e35036584a952625d51a2 and 🍒-picking f93fe728c2616505a0b663f685af54c20de52398 (or just wait for its merge, and update from trunk).

On the second thought - possibly keeping everything as it is will also work and cause no conflicts 🤔

Reply:
Ah I wasn't aware of that PR, thanks for linking that! I think I'll wait for that and update from trunk. I would still need to update the version number for `ScreenObject` package on this PR later.